spec/tapioca/addon_spec.rb

```diff
@@ -0,0 +1,59 @@
+# typed: true
+# frozen_string_literal: true
+
+require "addon_spec_helper"
+require "ruby_lsp/ruby_lsp_rails/runner_client"
+require "minitest/hooks"
+
+module RubyLsp
+  module Tapioca
+    class AddonSpec < Minitest::HooksSpec
+      # The approach here is based on tests within the Ruby LSP Rails gem
+
+      # TODO: Replace by `before(:all)` once Sorbet understands it
+      def initialize(*args)
+        super(*T.unsafe(args))
+        FileUtils.cp("spec/dummy/bin/rails", "bin/rails")
+        @outgoing_queue = Thread::Queue.new
+        @client = T.let(nil, T.nilable(RubyLsp::Rails::RunnerClient))
+        FileUtils.chdir("spec/dummy") do
```

Comment:
If we change the directory to boot the client here, why do we need to copy the `bin/rails` file above? Wouldn't the chdir already deal with that?

Reply:
Leftover from a previous approach, now removed.